tasks/managed/push-rpm-to-koji/README.md

```diff
@@ -7,6 +7,7 @@ Tekton task to push konflux build rpms to koji instance.
 | Name                    | Description                                                                                                                                                                | Optional | Default value        |
 |-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|----------------------|
 | snapshotPath            | Path to the JSON file of the mapped Snapshot spec in the data workspace                                                                                                    | No       | -                    |
+| releasePipelineMetadata | json object containing git resolver metadata about the running release pipeline                                                                                            | No       | -                    |
```

Comment:
Wouldn't it be better to provide separate much simpler string parameters instead? From the code below, it seems that only the build pipeline run name and namespace are required.